tests/console/command_not_found.pm

```diff
@@ -16,6 +16,9 @@ use strict;
 sub run() {
     # permissions don't matter
 
+    # ensure command-not-found is installed (it is not part of minimal text mode installs for example)
```

Comment:
Agreed - I was actually thinking if it's worth to do that, but then thought we test this at one point only anyway (extra_tests)... but I'll extend it to install only in textmode (minimalX is already contains enhanced_base pattern, and thus has it installed)